zkstack_cli/crates/zkstack/src/commands/ecosystem/common.rs

```diff
@@ -80,3 +181,150 @@ pub async fn deploy_l1(
 
     Ok(contracts_config)
 }
+
+#[allow(clippy::too_many_arguments)]
+pub async fn register_ctm_on_existing_bh(
+    shell: &Shell,
+    forge_args: &ForgeScriptArgs,
+    config: &EcosystemConfig,
+    l1_rpc_url: &str,
+    sender: Option<String>,
+    broadcast: bool,
+) -> anyhow::Result<()> {
+    let wallets_config = config.get_wallets()?;
+
+    let mut forge = Forge::new(&config.path_to_l1_foundry())
+        .script(&REGISTER_CTM_SCRIPT_PARAMS.script(), forge_args.clone())
+        .with_ffi()
+        .with_rpc_url(l1_rpc_url.to_string());
+
+    if config.l1_network == L1Network::Localhost {
+        // It's a kludge for reth, just because it doesn't behave properly with large amount of txs
+        forge = forge.with_slow();
+    }
+
+    if let Some(address) = sender {
+        forge = forge.with_sender(address);
+    } else {
+        // forge =
+        // fill_forge_private_key(forge, Some(&wallets_config.governor), WalletOwner::Governor)?;
+        forge = fill_forge_private_key(
```

Comment:
could you please add a comment why we need deployer wallet when usually only governor can register new ctms